### DISCUSSION TITLE
Lock button flyout bug

### DISCUSSION THREAD

**Discussion by Jaknator** [ShankarBUS/ModernFlyouts#954] [Q&A]
I wanted to ask whether somebody could help me with this bug. When, for example, I turn on the num lock, both the modern flyout version of the popup and the standarf windows versino show up. I dont know how to fix that. Does someone have a clue what might be going on here?
I hope the picture I inserted explains it a bit further. 
![image](https://user-images.githubusercontent.com/63076049/154343006-087666d5-0ee4-4027-8a0f-5dfd55d8045d.png)

**Comment by prakharb5** [comment]
The "standard" flyout you mention is from the OEM of your device, the manufacture (Lenovo, etc). Windows itself does not include a flyout for number lock.

Have a look around the OEM software you have.
